pkg/cosign/tlog.go

```diff
@@ -84,9 +84,17 @@ func getLogID(pub crypto.PublicKey) (string, error) {
 }
 
 func intotoEntry(ctx context.Context, signature, pubKey []byte) (models.ProposedEntry, error) {
+	var pubKeyBytes [][]byte
+
+	if len(pubKey) == 0 {
+		return nil, errors.New("none of the Rekor public keys have been found")
+	}
+
+	pubKeyBytes = append(pubKeyBytes, pubKey)
+
 	return types.NewProposedEntry(ctx, intoto.KIND, intoto_v001.APIVERSION, types.ArtifactProperties{
 		ArtifactBytes:  signature,
-		PublicKeyBytes: pubKey,
+		PublicKeyBytes: pubKeyBytes,
```

Comment:
had to check up on why the `rekorEntry` function here that reconstructs hashedrekord types didn't need to be changed: that directly creates a v0.0.1 hashed rekord type, instead of relying on the general entry construction.